environment.yml

```diff
@@ -11,6 +11,6 @@ dependencies:
   # fiona is a transitive dependency which needs GDAL. so we install with conda
   # TODO: we shouldn't have to install all this geo stuff, so once we break the
   # dependency on `pudl` we should remove this too.
-  - fiona>=1.8
+  - fiona>=1.9.5
```

Comment:
I'm not 100% sure if this is necessary (though presumably something was broken at some point and that's why it got added). But we've changed our dependency management / specs quite a bit in the PUDL repo since then. Maybe try remove this entirely just to see if it works?

Reply:
Well, that didn't work!